mailpoet/assets/js/src/landingpage/footer.tsx

```diff
@@ -0,0 +1,22 @@
+import { MailPoet } from 'mailpoet';
+import { Button } from 'common';
+import { Heading } from 'common/typography/heading/heading';
+
+function Footer(): JSX.Element {
+  return (
+    <div className="mailpoet-content-center landing-footer">
+      <Heading level={4}> {MailPoet.I18n.t('readyToUseMailPoet')} </Heading>
+      <Button
+        onClick={() => {
+          window.location.href = window.mailpoet_welcome_wizard_url;
+        }}
+      >
+        {' '}
```

Comment:
As I understand this is to add white space, na? then isn't it a responsibility of the css styling?

Reply:
This ` {' '}` right?

Prettier formater adds that in.